src/main/java/org/cryptomator/common/vaults/Vault.java

```diff
@@ -104,6 +105,7 @@ public class Vault {
 		this.settings = settings;
 		this.showingStats = new SimpleBooleanProperty(false);
 		this.quickAccessEntry = new AtomicReference<>(null);
+		this.lastKnownKeyLoader = Bindings.createStringBinding(this::getLastKnownKeyLoader, vaultSettings.lastKnownKeyLoader);
```

Comment:
Unnecessary wrapping of a stringProperty inside a Binding.

The Binding just returns the Content of the Property, hence we can simply use the property.

Reply:
Actually, is this used anywhere? If not, remove it.